app/services/reporting/monthly_active_users_count_report.rb

```diff
@@ -0,0 +1,44 @@
+module Reporting
+  class MonthlyActiveUsersCountReport
+    attr_reader :report_date
+
+    def initialize(report_date = Time.zone.today)
+      @report_date = report_date
+    end
+
+    def monthly_active_users_count_report
+      [
+        ['Monthly IAL1 Active', 'Monthly IAL2 Active', 'Total'],
+        [total_ial1_active, total_ial2_active, total_ial1_active + total_ial2_active],
```

Comment:
we try not to use "IAL2" in label names anymore, we usually call that IDV

WDYT of organizing the table as rows like this, and maybe including the month name in there as the title? So it gets rendered like this

##  September 2023 Active Users

| Monthly Active Users | Value |
| --- | --- |
| IAL1 | 100 |
| IDV | 1 |
| Total | 101 |

Reply:
Thats looks good and more informative, i will change this like this.